### ISSUE TITLE
Minirocket giving different test accuracies with every model on the same test data

### ISSUE THREAD

**Issue by hailthedawn** [timeseriesAI/tsai#745]
I am using the [MiniRocket classifier,](https://colab.research.google.com/github/timeseriesAI/tsai/blob/master/tutorial_nbs/10_Time_Series_Classification_and_Regression_with_MiniRocket.ipynb#scrollTo=cj2EXqZJHitT) and trying to perform emotion detection on ~1700 utterances. I've segmented the data into train and test, with about 300-400 utterances in the test set. Each utterance corresponds to an output label (of which there are three total).
Every time I run the build model -> get test accuracy cells, I'm getting very different test accuracies (ranging from 30% accuracy to 80%). What could be the reason behind this? I know my dataset is small - is that the reason? What do you recommend I do to get more consistencies with MiniRocket? 

Note: What I am actually providing as input to the classifier is CNN embeddings corresponding to each utterance.

**Comment by oguiza** [comment]
Hi @hailthedawn,
That's strange. I haven't seen that type of variation in the score.
Are you using something similar to this?
```python
from tsai.models.MINIROCKET import MiniRocketClassifier

# Univariate classification with sklearn-type API
dsid = 'OliveOil'
X_train, y_train, X_valid, y_valid = get_UCR_data(dsid)   # Download the UCR dataset

# Computes MiniRocket features using the original (non-PyTorch) MiniRocket code.
# It then sends them to a sklearn's RidgeClassifier (linear classifier).
model = MiniRocketClassifier()
timer.start(False)
model.fit(X_train, y_train)
t = timer.stop()
print(f'valid accuracy    : {model.score(X_valid, y_valid):.3%} time: {t}')

```
